container_files/public_html/doc/builtin/procedures/DistTableProcedure.md

```diff
@@ -0,0 +1,75 @@
+# Stats Table Procedure
+
+For a combination of feature columns (the `training_data` parameter) and outcomes, this procedure
+will iteratively (over the rows) calculate different statistics (count, avg, std, min, max) on
+the outcomes, for each values of the feature columns. 
+
+The feature columns are assumed to be string-valued, and the outcome numeric-valued (type `INTEGER` or `NUMBER`). 
+
+This procedure can be views as a ![](%%doclink statsTable.bagOfWords.train procedure) where the
```

Comment:
Faut linker les stats tables normales et pas les bag of words non? genre tes colones sont fixes en input, contrairement au bag of words qui sont sparse